reference.jsonc

```diff
@@ -0,0 +1,84 @@
+{
+    "errors": [
+        // Graph creation
```

Comment:
Is this reference file just for us in the future or does it have some programmatic use?

Reply:
Yes I forgot to mention that. This is what I originally started with and used as reference as I kept going, thought it'd be good to include during the draft PR